assets/js/components/setup/compatibility-checks.test.js

```diff
@@ -84,6 +84,9 @@ describe( 'CompatibilityChecks', () => {
 	it( 'should make API requests to "setup-checks, health-checks and AMP Project test URL', async () => {
 		const token = 'test-token-value';
 
+		global._googlesitekitBaseData = global._googlesitekitBaseData || {};
+		global._googlesitekitBaseData[ 'isWP5.0+' ] = true;
```

Comment:
We should add the definition to `.storybook/config.js` as well.

Reply:
We don't need it anymore after I have reworked it using your suggestion.